Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ponylang/ponyup-ci-x86-64-unknown-linux-builder:release AS build
```

Comment:
i can't even find the Dockerfile for this. It's so old. And it never existed in this repo. Oooops.